bcgov/foi-flow
# Dev marshal st 3119

Verification of acceptance criteria

Unit Testing
====

- Create Request
- State Transition
- Verify document upload (records)
- Email for Fee
- Comments for request history
